### PR TITLE
DOCS-1089: Update Modular resource examples to use new UI

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/extend/modular-resources/examples/tflite-module.md [hunk 1]
```diff
@@ -228,7 +228,12 @@ To generate your robot's configuration using `example_audio_classification_clien
    ```
 
 1. Copy the contents of this file.
-   Then return to your robot's page on [the Viam app](https://app.viam.com), select the **Config** tab, select **Raw JSON**, and paste the configuration into the text area.
+   Then return to your robot's page on [the Viam app](https://app.viam.com), select the **Config** tab, select **Raw JSON**, and add the configuration into the text area.
```
Comment (npentrel) [inline docs/extend/modular-resources/examples/tflite-module.md:231]: @andf-viam can you confirm this is a correct addition?

File: docs/extend/modular-resources/examples/csi.md [hunk 2]
```diff
@@ -32,40 +31,62 @@ sudo chmod 755 /usr/local/bin/viam-csi
 ## Configuration
 
 {{< tabs name="Connect your CSI Module and Modular Resource">}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+
+Click on the **Modules** subtab and navigate to the **Local** section.
+Enter a name, for example `my_csi_cam_module_name`, and the executable path, for example `/usr/bin/csi-mr`.
```
Comment (andf-viam) [inline docs/extend/modular-resources/examples/csi.md:39]:
```suggestion
Enter a name, for example `my_csi_cam_module_name`, then enter the executable path.
If you used the above install command, the executable path should be: `/usr/local/bin/viam-csi`.
```

File: docs/extend/modular-resources/examples/csi.md [hunk 3]
```diff
@@ -32,40 +31,62 @@ sudo chmod 755 /usr/local/bin/viam-csi
...
+Then click **Add module**.
+
+Click on the **Components** subtab and click **Create component**.
+Select the `local modular resources` type.
```
Comment (andf-viam) [inline docs/extend/modular-resources/examples/csi.md:43]:
```suggestion
Select the `local modular resource` type.
```
To match UI

File: docs/extend/modular-resources/examples/csi.md [hunk 4]
```diff
@@ -32,40 +31,62 @@ sudo chmod 755 /usr/local/bin/viam-csi
...
-```json {class="line-numbers linkable-line-numbers"}
+```json
 {
-    "modules": [
-      {
-        "executable_path": "</usr/bin/csi-mr>",
-        "name": "<your-csi-cam-module-name>"
-      }
-    ]
+    "executable_path": "</usr/bin/csi-mr>",
```
Comment (andf-viam) [inline docs/extend/modular-resources/examples/csi.md:68]:
```suggestion
    "executable_path": "</usr/local/bin/viam-csi>",
```

File: docs/extend/modular-resources/examples/odrive.md [hunk 5]
```diff
@@ -93,82 +93,84 @@ Plug a USB-C to USB-A cable from the isolator to the ODrive.
...
+Click on the **Modules** subtab and navigate to the **Local** section.
+Enter a name, for example `my_odrive_motor`, and the executable path, that points to the location where your ODrive module’s run script, <file>run.sh</file>, is stored on your robot’s computer.
+Then click **Add module**.
 
 ![The ODrive module with the name 'odrive' and executable path '~/desktop/odrive/odrive-motor/run.sh' added to a robot in the Viam app config builder](/extend/modular-resources/add-odrive/add-odrive-module-ui.png)
 
-{{% /tab %}}
-{{% tab name="JSON Template" %}}
+Click on the **Components** subtab and click **Create component**.
+Select the `local modular resources` type.
```
Comment (andf-viam) [inline docs/extend/modular-resources/examples/odrive.md:122]:
```suggestion
Select the `local modular resource` type.
```

File: docs/extend/modular-resources/examples/odrive.md [hunk 6]
```diff
@@ -93,82 +93,84 @@ Plug a USB-C to USB-A cable from the isolator to the ODrive.
...
+Then select the `camera` as the type, enter the triplet `viam:camera:csi` and give your resource a name, for example `my_test_csi_cam`.
```
Comment (andf-viam) [inline docs/extend/modular-resources/examples/odrive.md:123]:
```suggestion
Then select `motor` as the type, enter the triplet `viam:odrive:serial`, and give your resource a name, for example `my_test_odrive`.
```

File: docs/extend/modular-resources/examples/csi.md [hunk 7]
```diff
@@ -32,40 +31,62 @@ sudo chmod 755 /usr/local/bin/viam-csi
...
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+
+Click on the **Modules** subtab and navigate to the **Local** section.
+Enter a name, for example `my_csi_cam_module_name`, and the executable path, for example `/usr/bin/csi-mr`.
+Then click **Add module**.
+
+Click on the **Components** subtab and click **Create component**.
+Select the `local modular resources` type.
+Then select the `camera` as the type, enter the triplet `viam:camera:csi` and give your resource a name, for example `my_test_csi_cam`.
```
Comment (andf-viam) [inline docs/extend/modular-resources/examples/csi.md:44]:
```suggestion
Then select `camera` as the type, enter the triplet `viam:camera:csi`, and give your resource a name, for example `my_test_csi_cam`.
```

File: docs/extend/modular-resources/examples/tflite-module.md [hunk 8]
```diff
@@ -228,7 +228,12 @@ To generate your robot's configuration using `example_audio_classification_clien
...
+
+   {{< alert title="Important" color="note" >}}
+   If you already have other configured components, you will need to add each generated JSON object to the respective `modules` or `services` array.
+   If you do not already have cconfigured components, you can replace the contents in **Raw JSON** with the generated contents.
```
Comment (andf-viam) [inline docs/extend/modular-resources/examples/tflite-module.md:235]:
```suggestion
   If you do not already have configured components, you can replace the contents in **Raw JSON** with the generated contents.
```

**Review by andf-viam — APPROVED** [review]
So, changes look good, but I think the ticket ask was for adding the modules that are now in the Viam Registry to your robot directly from the registry, rather than as local modules. That's DOCS - 1080 for that effort. 

CSI and TFLite modules are not on the registry, so would indeed need these instructions (both look good to merge to me after some nit fixes), but I'll update ODrive and RPlidar, both on the Registry, through DOCS - 1080 after.

So: let's merge this, and I'll add Registry instructions for Odrive and RPlidar in DOCS - 1080 as well.

**Comment by andf-viam** [comment]
Gonna merge this in, and base my DOCS - 1080  work on this! Thanks!

**Pull request merged**